frontend/src/components/auth/Login.tsx

```diff
@@ -25,17 +40,24 @@ const Login = (): React.ReactElement => {
   const { authenticatedUser, setAuthenticatedUser } = useContext(AuthContext);
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
+  const [isSecondPage, setIsSecondPage] = useState<boolean>(false);
   const history = useHistory();
 
   const [login] = useMutation<{ login: AuthenticatedUser }>(LOGIN);
 
   const onLogInClick = async () => {
-    const user: AuthenticatedUser = await authAPIClient.login(
-      email,
-      password,
-      login,
-    );
-    setAuthenticatedUser(user);
+    if (isSecondPage === false) {
+      setIsSecondPage(true);
+    } else if (isSecondPage === true) {
+      const user: AuthenticatedUser = await authAPIClient.login(
+        email,
+        password,
+        login,
+      );
+      setAuthenticatedUser(user);
+    } else {
```

Comment:
I think since we have two separate button components, it'd be great two separate the logic into two functions, one for each button. That we we can avoid this conditional statements and just run what we want based on the buttons